ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/TransactionServiceVisibilityIT.scala

```diff
@@ -28,9 +28,21 @@ import com.daml.test.evidence.tag.Security.SecurityTest.Property.Privacy
 
 import scala.collection.immutable.Seq
 import scala.collection.mutable
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration.DurationInt
 
 class TransactionServiceVisibilityIT extends LedgerTestSuite {
 
+  // triple eventually wait duration compared to default to avoid database timeouts
+  // when running against Oracle in enterprise mode
+  def eventually[A](
+      assertionName: String
+  )(runAssertion: => Future[A])(implicit ec: ExecutionContext): Future[A] =
+    Eventually.eventually(
+      assertionName,
+      firstWaitTime = 30.millis, // compared to default of 10.millis
```

Comment:
how about increasing attemps from 10 to 12 for times 4 maximum waiting times instead? (to keep it same reactive, but retrying longer)

Reply:
Good idea.